src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,56 @@
+@import "../utils/variables.scss";
+
+.calendar {
+  width: 727px;
+  display: flex;
+  gap: $calendar-cell-gap;
+  flex-wrap: wrap;
+  padding: 20px;
+
+  &--start-day {
+    $start-days:
+      "mon" 0 * ($calendar-cell-width + $calendar-cell-gap),
+      "tue" 1 * ($calendar-cell-width + $calendar-cell-gap),
+      "wed" 2 * ($calendar-cell-width + $calendar-cell-gap),
+      "thu" 3 * ($calendar-cell-width + $calendar-cell-gap),
+      "fri" 4 * ($calendar-cell-width + $calendar-cell-gap),
+      "sat" 5 * ($calendar-cell-width + $calendar-cell-gap),
+      "sun" 6 * ($calendar-cell-width + $calendar-cell-gap);
+
+    @each $day, $start-day in $start-days {
+      &--#{$day} {
```

Comment:
You should apply this only to :first-child of the calendar, not the whole calendar
```scss
@each $day, $start-day in $start-days {
      &--#{$day} :first-child {
```